classes/Forms.js

```diff
@@ -8,14 +8,14 @@ const OneBlinkAPI = require('../lib/one-blink-api.js')
 const setPreFillData = require('../lib/pre-fill-data')
 const { validateWithFormSchema } = require('../lib/forms-validation.js')
 const generateFormElement = require('../lib/generate-form-element.js')
-const getTenantUrl = require('../lib/tenant')
 module.exports = class Forms extends OneBlinkAPI {
   constructor(options /* : ConstructorOptions */) {
     options = options || {}
     super(
-      getTenantUrl(options.tenant, options.oneBlinkAPIOrigin),
+      options.oneBlinkAPIOrigin,
```

Comment:
Can't remove this now? And from all the other class constructors?

Reply:
Didn't put the env var feature in yet